buildroot/package/m4/m4.mk

```diff
@@ -4,10 +4,10 @@
 #
 ################################################################################
 
-M4_VERSION = 1.4.17
+M4_VERSION = 1.4.18
 M4_SOURCE = m4-$(M4_VERSION).tar.xz
 M4_SITE = $(BR2_GNU_MIRROR)/m4
-M4_LICENSE = GPLv3+
+M4_LICENSE = GPLv3.0+
```

Comment:
Hmmm, why has `BISON_LICENSE` changed from `GPLv3+` to `GPL-3.0+`, but `M4_LICENSE` changed from `GPLv3+` to `GPLv3.0+` ?

Hmmmm part 2: https://buildroot.uclibc.org/downloads/manual/manual.html#_infrastructure_for_packages_with_specific_build_systems seems to suggest that the "SPDX short identifier" should be used, and the SPDX page that it links to lists `GPL-3.0+` under the "Deprecated License Identifiers" section :confused: :man_shrugging: 

Looks like "it's complicated" so feel free to ignore this comment if you want :wink:

Reply:
A lot of the changes were lifted from other buildroots (like berryboot's version of buildroot, just copying and pasting) and I didn't really work to ensure that the makefile strings were completely correct while copying them, so I can't attest that they are all correct. I was just trying to get the project compiling tbh, so I was working quick any dirty (and it was past 3-4am by the time I finally got this compiling).

Reply:
> it was past 3-4am by the time I finally got this compiling

Been there, done that... :joy: